examples/basic.js

```diff
@@ -23,20 +23,11 @@ var SplunkLogger = require("../index").Logger;
 
 /**
  * Only the token property is required.
- * Defaults are listed explicitly.
- *
- * Alternatively, specify config.url like so:
- *
- * "https://localhost:8088/services/collector/event/1.0"
  */
 var config = {
     token: "your-token-here",
-    host: "localhost",
-    path: "/services/collector/event/1.0",
-    protocol: "https",
-    port: 8088,
-    level: "info",
-    autoFlush: true
+    url: "https://localhost:8088",
+    maxBatchCount: 1 // Send events 1 at a time
```

Comment:
If none of the batch settings are configured, and `autoFlush === true` (the default) it's basically not manual batching